onboarding/src/pages/UserPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Header } from '../component/Header';
+import { useGetUserById } from '../hooks/useGetUserById';
+
+export const UserPage: React.FC = () => {
+  const params: { id: string | undefined } = useParams();
```

Comment:
Aqui o lint reclama disso?
```suggestion
  const params: { id?: string } = useParams();
```

Reply:
incrivelmente não reclamou, mas gostei da sugestão =)